win32/select.c

```diff
@@ -82,6 +85,7 @@ PHPAPI int php_select(php_socket_t max_fd, fd_set *rfds, fd_set *wfds, fd_set *e
 					sock_max_fd = i;
 				}
 			} else {
+				handles[n_handles] = (HANDLE)(uintptr_t)_get_osfhandle(i);
```

Comment:
I wonder wether an `INVALID_HANDLE_VALUE` can be returned here, and what to do in this case. Previously, that would have been treated as socket. Maybe we should stick with this; otherwise, it might be prudent to ignore such invalid handles.

Reply:
Good catch, I hadn't considered that. I'm doubtful assuming it's a socket is a good idea since that might cause issues with socket-specific parts of the code. Probably best to just skip them.